scripts/populate_lambda_ssm_parameters.py

```diff
@@ -102,12 +142,21 @@ def get_admin_user_emails():
         default=None,
         help="Remove a single environment variable in SSM parameters and all deployed lambdas"
     )
+    parser.add_argument("--verify",
+        nargs='*',
+        help='Verify the deployed Lambda Environment Variables for a current stage'
+    )
     args = parser.parse_args()
 
     if args.print:
         ssm_env = get_ssm_lambda_environment()
         for name, val in ssm_env.items():
             print(f"{name}={val}")
+    elif args.verify:
+        print(f"Deployment Stage: {os.getenv('DSS_DEPLOYMENT_STAGE')}")
```

Comment:
Should include something more descriptive like `Verifying env vars in deployment stage: `.